src/io/output_time.cc

```diff
@@ -212,6 +256,9 @@ void OutputTime::write_time_frame()
 			// Remember the last time of writing to output stream
 			write_time = time;
 			current_step++;
+            
+            // unset validity of the output mesh after data are written at the current write time
+            is_output_mesh_valid_ = false;
```

Comment:
Sorry, not for flow and transport..
In case there are some reaction terms, then the output mesh would be constructed separately for sorption and dual porosity, since there are more field sets wrtiting into the same output stream...